packages/account-abstraction/src/BiconomyPaymasterAPI.ts

```diff
@@ -15,7 +15,10 @@ export class BiconomyPaymasterAPI implements IPaymasterAPI {
     this.paymasterConfig = paymasterConfig
   }
 
-  async getPaymasterAndData(userOp: Partial<UserOperation>): Promise<string> {
+  async getPaymasterAndData(
+    userOp: Partial<UserOperation>,
+    paymasterServiceData?: object
```

Comment:
IPaymasterAPI also needs to change to accept custom parameters other than userOp...

Reply:
Should we do that in this PR? Or we when the SDK is being modular?

Reply:
I was thinking the build would fail cause this class currently implements the mentioned interface

Reply:
Yes the changes are reflected in IPaymaster also
https://github.com/bcnmy/biconomy-client-sdk/blob/fae8b3a02a5e810a9a40674d27f389b89199bb62/packages/core-types/src/AccountAbstractionTypes.ts#L4